src/quo/foundations/colors.cljs

```diff
@@ -250,6 +250,25 @@
    :polygon  "#AD71F3"
    :unknown  "#EEF2F5"})
 
+(def socials
+  {:link      "#647084"
+   :facebook  "#1877F2"
+   :github    "#000000"
+   :instagram "#D8408E0F"
+   :lens      "#00501E"
+   :linkedin  "#0B86CA"
+   :mirror    "#3E7EF7"
+   :opensea   "#2081E2"
+   :pinterest "#CB2027"
+   :rarible   "#FEDA03"
+   :snapchat  "#FFFC00"
+   :spotify   "#00DA5A"
+   :superrare "#000000"
+   :tumblr    "#37474F"
+   :twitch    "#673AB7"
+   :twitter   "#262E35"
+   :youtube   "#FF3000"})
```

Comment:
Since these will be used by `resolve-color`, makes sense that it they are prefixed by `:social` or something like that?
example:

`(resolve-color :social/link :light)`
or
`(resolve-color :social/youtube :dark)`

I'm not sure tbh, but I'd like to know if you have it already considered.
cc: @J-Son89

Reply:
I had not consider it, but I think it could be nice to do that way 👍 migth be nice to do for the other colors too? 
No strong opinions about it really though as we have not done that for other colors, e.g :custom/ and :network/

Reply:
Yes! I think that'd be awesome!

The code will be a little more verbose (but it already is!), and IMO the reading will be improved a lot.
We should get more feedback from other devs, but I completely support it!

Reply:
Having meaningful keywords and using their capability to handle namespaces improves the reading/refactoring/code-suggestion a lot.

Reply:
I added `:social/` prefix to the colors

Reply:
The colors are the same in the dark and light mode